app/controllers/projects_controller.rb

```diff
@@ -2,16 +2,23 @@ class ProjectsController < ApplicationController
   def user_project_metric
     return if metric_params.blank?
 
-    @metrics = Queries::BaseQueryMetric.determinate_metric_period(metric_params[:period]).call(
-      project_id: project_id,
-      metric_name: metric_params[:name]
-    )
+    period = metric_params[:period]
+    @review_turnaround = Queries::BaseQueryMetric.determinate_metric_period(period)
+                                                 .call(
+                                                   project_id: project_id,
+                                                   metric_name: 'review_turnaround'
+                                                 )
+    @merge_time = Queries::BaseQueryMetric.determinate_metric_period(period)
```

Comment:
Are we calling `determinate_metric_period` everytime we use the `call` method? If so, perhaps that code can go inside `call`.

Reply:
roger that